L/LLVMExtra/build_tarballs.jl

```diff
@@ -4,16 +4,16 @@ using Base.BinaryPlatforms
 include("../../fancy_toys.jl")
 
 name = "LLVMExtra"
-repo = "https://github.com/vchuravy/LLVMExtra.git"
-version = v"0.0.1"
+repo = "https://github.com/maleadt/LLVM.git"
+version = v"0.0.2"
 
 # Collection of sources required to build attr
-sources = [GitSource(repo, "0f9293476db7bc60268dfa041acbdef82387861a")]
+sources = [GitSource(repo, "93476db7bc60268dfa041acbdef82387861a")]
```

Comment:
Should we ensure the sha is valid after merging the PR then? I did rebase, so it w643d need to be e.g. https://github.com/maleadt/LLVM.jl/pull/246/commits/83e890b2ed146a3275be04425ee0e1193c1a1da9 (and merge, not squash the PR)*

Reply:
YEah that would be best :) (Or at leastt keep it stable until the build is through xD)